keyboards/niu_mini/keymaps/shurikai/config.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Done.

Reply:
Should I change headers in other files as well?

Reply:
Does not look like it took.  Did you push the changes

Reply:
> Should I change headers in other files as well?

Only on c and h files. rules.mk don't need them